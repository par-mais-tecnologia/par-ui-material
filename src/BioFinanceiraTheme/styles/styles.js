import './animations.css'
import './colors.css'
import './fonts.css'
import './reset.css'

const colors = {
  gray_01: '#636363',
  gray_02: '#9C9C9C',
  gray_03: '#D4D4D4',
  gray_04: '#F0F0F0',

  fairy: '#B6DDDC',
  aqua: '#8ACAC7',
  turquise: '#5EB8C0',
  siren: '#019193',
  see: '#347A7C',

  pink: '#D4436B',

  green: '#94BA1D',

  purple: '#632B7D',

  cobalt: '#007CA3',

  white: '#ffffff',

  seeGradient: 'linear-gradient(26deg, #347A7C, #8ACAC7)',
  bgFooter: '#383838'
}

const classes = {
  title1: 'f4 brandon-regular pink',
  title2: 'f1 roboto-medium gray_01',
  title3: 'f1 roboto-medium turquise',
  subtitle1: 'f3 roboto-light gray_01',
  subtitle2: 'f3 roboto-medium turquise',
  subtitle3: 'f4 brandon-regular turquise',
  subtitle4: 'f6 brandon-bold turquise',
  subtitle5: 'f7 roboto-light gray_04',
  body1: 'f5 roboto-light gray_01',
  body2: 'f4 roboto-light gray_01',
  body4: 'f5 roboto-medium gray_01',
  highlight1: 'f5 roboto-medium pink',
  highlight2: 'f6 roboto-regular gray_01',
  highlight3: 'f5 roboto-light pink',
  links: 'f6 roboto-bold gray_02',
  button: 'f6 roboto-medium pink toUppercase'
}

const fonts = {
  title1: {
    fontFamily: '\'Brandon Regular\', sans-serif',
    fontSize: '1.25rem',
    color: colors.pink
  },
  title2: {
    fontFamily: '\'Roboto Medium\', sans-serif',
    fontSize: '3rem',
    color: colors.gray_01
  },
  title3: {
    fontFamily: '\'Roboto Medium\', sans-serif',
    fontSize: '3rem',
    color: colors.turquise
  },
  subtitle1: {
    fontFamily: '\'Roboto Light\', sans-serif',
    fontSize: '1.5rem',
    color: colors.gray_01
  },
  subtitle2: {
    fontFamily: '\'Roboto Medium\', sans-serif',
    fontSize: '1.5rem',
    color: colors.turquise
  },
  subtitle3: {
    fontFamily: '\'Roboto Regular\', sans-serif',
    fontSize: '1.25rem',
    color: colors.turquise
  },
  subtitle4: {
    fontFamily: '\'Roboto Regular\', sans-serif',
    fontSize: '.875rem',
    color: colors.turquise
  },
  body1: {
    fontFamily: '\'Roboto Light\', sans-serif',
    fontSize: '1rem',
    color: colors.gray_01
  },
  body2: {
    fontFamily: '\'Roboto Regular\', sans-serif',
    fontSize: '.875rem',
    color: colors.gray_02
  },
  body3: {
    fontFamily: '\'Roboto Medium\', sans-serif',
    fontSize: '3rem',
    color: colors.turquise
  },
  body4: {
    fontFamily: '\'Roboto Medium\', sans-serif',
    fontSize: '1rem',
    color: colors.gray_01
  },
  highlight1: {
    fontFamily: '\'Roboto Medium\', sans-serif',
    fontSize: '1rem',
    color: colors.pink
  },
  highlight2: {
    fontFamily: '\'Roboto Regular\', sans-serif',
    fontSize: '.875rem',
    color: colors.gray_01
  },
  highlight3: {
    fontFamily: '\'Roboto Light\', sans-serif',
    fontSize: '1rem',
    color: colors.pink
  },
  links: {
    fontFamily: '\'Brandon Regular\', sans-serif',
    fontSize: '1.25rem',
    color: colors.pink
  },
  button: {
    fontFamily: '\'Roboto Medium\', sans-serif',
    fontSize: '.875rem',
    color: colors.pink
  },
  subheading: {
    fontFamily: '\'Roboto Medium\', sans-serif',
    fontSize: '1rem',
    color: colors.pink
  },
  mainTitle: {
    fontFamily: "'Brandon Light', sans-serif",
    fontSize: '1.6rem',
    color: colors.white
  },
  footerText: {
    fontFamily: 'Roboto-Light, sans-serif',
    fontSize: '0.875rem',
    color: colors.gray_04
  },
  headline: {
    fontFamily: 'Roboto-Light, sans-serif',
    fontSize: '1.25rem',
    color: colors.gray_01
  }
}

export const styles = {
  colors,
  classes,
  fonts
}
