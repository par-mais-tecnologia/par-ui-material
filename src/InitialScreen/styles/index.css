.img-bg {
    background-position-x: center;
}

.emailInput input {
    color: #ffffff
}

.passwordInput input {
    color: #ffffff
}