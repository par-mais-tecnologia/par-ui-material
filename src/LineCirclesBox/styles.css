.line-circles-box {
    padding-left: 15px;
    position: relative;
}
.line-circles-box:before,
.line-circles-box:after {
    position: absolute;
    bottom: -4px;
    left: -3PX;
    height: 5px;
    width: 5px;
    content: "";
    border-radius: 20px;
}
.line-circles-box:after {
    right: 0;
    top: -5px;
}
