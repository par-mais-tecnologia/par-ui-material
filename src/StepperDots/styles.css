#nextIcon {
  transform: rotate(90deg) scale(0.5);
}

#backIcon {
  transform: rotate(270deg) scale(0.5);
}