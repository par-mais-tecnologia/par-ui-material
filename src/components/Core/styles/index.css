@import '_fonts.css';

/* Fonts */
.brandon-light {
    font-family: 'Brandon-Light', sans-serif;
}

.brandon-regular {
    font-family: 'Brandon Regular', sans-serif;
}

.brandon-bold {
    font-family: 'Brandon Bold', sans-serif;
}

.roboto-light {
    font-family: 'Roboto Light', sans-serif;
}

.roboto-medium {
    font-family: 'Roboto Medium', sans-serif;
}

.roboto-bold {
    font-family: 'Roboto Bold', sans-serif;
}

.rounded-elegance {
    font-family: 'Rounded Elegance', sans-serif;
}
