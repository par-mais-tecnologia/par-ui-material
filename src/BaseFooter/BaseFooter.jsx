import React, { PureComponent } from 'react'
import Grid from '@material-ui/core/Grid'
import { withStyles } from '@material-ui/core/styles'

import PropTypes from 'prop-types'

const styles = (theme) => ({
  footer: {
    position: 'fixed',
    bottom: '0px',
    width: ' 100%',
    margin: '0 auto',
    background: theme.styles && theme.styles.bgFooter ? theme.styles.bgFooter : '#000',
    zIndex: ' 100'
  }
})

class BaseFooter extends PureComponent {
  render () {
    const { children, classes, height } = this.props

    return (
      <Grid container justify='center' style={{ height }} className={classes.footer}>
        {children}
      </Grid>
    )
  }
}

BaseFooter.propTypes = {
  children: PropTypes.oneOfType([
    PropTypes.arrayOf(PropTypes.node),
    PropTypes.node
  ]).isRequired,
  height: PropTypes.number.isRequired
}

export default withStyles(styles)(BaseFooter)
