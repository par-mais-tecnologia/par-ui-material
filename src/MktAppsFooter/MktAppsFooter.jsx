import React, { PureComponent } from 'react'
import Grid from '@material-ui/core/Grid'
import { withStyles } from '@material-ui/core/styles'

const styles = theme => {
  const { styles } = theme
  return {
    title1: {
      ...theme.typography.body2,
      color: styles.colors.fairy,
      lineHeight: 1.2
    },
    title2: {
      ...theme.typography.body2,
      color: styles.colors.gray_04,
      lineHeight: 1.2
    }
  }
}

class MktAppsFooter extends PureComponent {
  render () {
    const { classes } = this.props

    return (
      <Grid container justify='space-around' className='pa3'>
        <Grid item className='flex w-100 w-auto-l pb2 items-center justify-center items-center'>
          <img src='https://static.parmais.com.br/images/parmais-logo.svg' width='100px' />
        </Grid>
        <Grid className='flex flex-wrap tc tl-l mb3 mb0-l items-center'>
          <Grid className={`${classes.title1} w-100 w-auto-l pb2 pb0-l mb2 mb0-l bb b--white-20 bn-l`}>
            <span className='di db-l'>Par Mais</span>
            <span> Emponderamento Financeiro</span>
          </Grid>
          <Grid className='bl-l b--white-20 ml3-l pl3-l w-100 w-auto-l mt1 mt0-l'>
            <span className={classes.title2}>
                Par Mais gestão administração de valores imobiliários <br />
                CNPJ: 21.719.643/0001-60
            </span>
          </Grid>
        </Grid>
        <Grid className='flex w-100 w-auto-l justify-center items-center'>
          <span className={`${classes.title1} dn db-l`}>
              Nossas<br />redes sociais
          </span>
          <Grid className='bl-l b--white-20 ml0 ml3-l'>
            <a href=''>
              <i className='ml0 ml3-l white par-icon-facebook' />
            </a>
            <a href=''>
              <i className='ml3 ml2-l white par-icon-linkedin' />
            </a>
            <a href=''>
              <i className='ml3 ml2-l white par-icon-youtube' />
            </a>
            <a href=''>
              <i className='ml3 ml2-l white par-icon-instagram' />
            </a>
          </Grid>
        </Grid>
      </Grid>
    )
  }
}

export default withStyles(styles, { withTheme: true })(MktAppsFooter)
