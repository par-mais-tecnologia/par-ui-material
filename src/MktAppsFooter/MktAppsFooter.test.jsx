import React from 'react'
import { configure, shallow } from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'
import { MuiThemeProvider } from '@material-ui/core/styles'

import finBioTheme from '../../src/BioFinanceiraTheme'
import BaseFooter from './'
import MktAppsFooter from './MktAppsFooter'

configure({ adapter: new Adapter() })

const getShallow = () => (
  <MuiThemeProvider theme={finBioTheme}>
    <BaseFooter>
      <MktAppsFooter />
    </BaseFooter>
  </MuiThemeProvider>
)

describe('mktAppsFooter', () => {
  const wrapper = shallow(getShallow())

  it('shallow mktAppsFooter', () => {
    expect(wrapper).toBeTruthy()
  })

  it('shallow mktAppsFooter match Snapshot', () => {
    expect(wrapper).toMatchSnapshot()
  })
})
