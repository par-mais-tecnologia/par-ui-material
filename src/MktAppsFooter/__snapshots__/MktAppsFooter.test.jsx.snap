// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`mktAppsFooter shallow mktAppsFooter match Snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <MuiThemeProvider
    theme={
      Object {
        "breakpoints": Object {
          "between": [Function],
          "down": [Function],
          "keys": Array [
            "xs",
            "sm",
            "md",
            "lg",
            "xl",
          ],
          "only": [Function],
          "up": [Function],
          "values": Object {
            "lg": 1280,
            "md": 960,
            "sm": 600,
            "xl": 1920,
            "xs": 0,
          },
          "width": [Function],
        },
        "direction": "ltr",
        "mixins": Object {
          "gutters": [Function],
          "toolbar": Object {
            "@media (min-width:0px) and (orientation: landscape)": Object {
              "minHeight": 48,
            },
            "@media (min-width:600px)": Object {
              "minHeight": 64,
            },
            "minHeight": 56,
          },
        },
        "overrides": Object {
          "MuiFormHelperText": Object {
            "required": Object {
              "color": "#f44336",
            },
          },
          "MuiFormLabel": Object {
            "asterisk": Object {
              "display": "none",
            },
          },
          "MuiInputBase": Object {
            "input": Object {
              "fontFamily": "Roboto Light",
            },
          },
          "MuiInputLabel": Object {
            "root": Object {
              "fontFamily": "Roboto",
            },
          },
          "MuiPickersSlideTransition": Object {
            "transitionContainer": Object {
              "fontFamily": "Roboto Light",
              "fontSize": "12px",
            },
          },
          "MuiSlider": Object {
            "thumb": Object {
              "backgroundColor": "white",
              "boxShadow": "1px 1px 20px #9C9C9C",
              "height": 30,
              "width": 30,
            },
            "track": Object {
              "background": "linear-gradient(to right, #8ACAC7 0%, #347A7C 100%)",
              "borderRadius": "10px",
              "boxShadow": "1px 1px 5px #9C9C9C",
              "height": "20px",
            },
            "trackAfter": Object {
              "background": "#c6c1c1",
            },
          },
        },
        "palette": Object {
          "action": Object {
            "active": "rgba(0, 0, 0, 0.54)",
            "disabled": "rgba(0, 0, 0, 0.26)",
            "disabledBackground": "rgba(0, 0, 0, 0.12)",
            "hover": "rgba(0, 0, 0, 0.08)",
            "hoverOpacity": 0.08,
            "selected": "rgba(0, 0, 0, 0.14)",
          },
          "augmentColor": [Function],
          "background": Object {
            "default": "#fafafa",
            "paper": "#fff",
          },
          "common": Object {
            "black": "#000",
            "white": "#fff",
          },
          "contrastThreshold": 3,
          "divider": "rgba(0, 0, 0, 0.12)",
          "error": Object {
            "contrastText": "#fff",
            "dark": "#d32f2f",
            "light": "#e57373",
            "main": "#f44336",
          },
          "getContrastText": [Function],
          "grey": Object {
            "100": "#f5f5f5",
            "200": "#eeeeee",
            "300": "#e0e0e0",
            "400": "#bdbdbd",
            "50": "#fafafa",
            "500": "#9e9e9e",
            "600": "#757575",
            "700": "#616161",
            "800": "#424242",
            "900": "#212121",
            "A100": "#d5d5d5",
            "A200": "#aaaaaa",
            "A400": "#303030",
            "A700": "#616161",
          },
          "primary": Object {
            "contrastText": "rgba(0, 0, 0, 0.87)",
            "dark": "rgb(65, 128, 134)",
            "light": "rgb(126, 198, 204)",
            "main": "#5EB8C0",
          },
          "secondary": Object {
            "contrastText": "#fff",
            "dark": "#c51162",
            "light": "#ff4081",
            "main": "#f50057",
          },
          "text": Object {
            "disabled": "rgba(0, 0, 0, 0.38)",
            "hint": "rgba(0, 0, 0, 0.38)",
            "primary": "rgba(0, 0, 0, 0.87)",
            "secondary": "rgba(0, 0, 0, 0.54)",
          },
          "tonalOffset": 0.2,
          "type": "light",
        },
        "patrimonyResultChart": Object {
          "&$patrimony .display4": Object {
            "color": "#707070",
            "fontFamily": "Roboto Medium",
          },
          "&$patrimony .percentage": Object {
            "color": "#A3A3A3",
          },
          "&$patrimony:hover .bar": Object {
            "animation": "createBox 0.2s forwards",
          },
          "&$patrimony:hover .onHover.investment": Object {
            "color": "#632B7D",
          },
          "&$patrimony:hover .onHover.movable": Object {
            "color": "#347A7C",
          },
          "&$patrimony:hover .onHover.realstate": Object {
            "color": "#5EB8C0",
          },
          "paddingBottom": "16px",
        },
        "props": Object {},
        "shadows": Array [
          "none",
          "0px 1px 3px 0px rgba(0, 0, 0, 0.2),0px 1px 1px 0px rgba(0, 0, 0, 0.14),0px 2px 1px -1px rgba(0, 0, 0, 0.12)",
          "0px 1px 5px 0px rgba(0, 0, 0, 0.2),0px 2px 2px 0px rgba(0, 0, 0, 0.14),0px 3px 1px -2px rgba(0, 0, 0, 0.12)",
          "0px 1px 8px 0px rgba(0, 0, 0, 0.2),0px 3px 4px 0px rgba(0, 0, 0, 0.14),0px 3px 3px -2px rgba(0, 0, 0, 0.12)",
          "0px 2px 4px -1px rgba(0, 0, 0, 0.2),0px 4px 5px 0px rgba(0, 0, 0, 0.14),0px 1px 10px 0px rgba(0, 0, 0, 0.12)",
          "0px 3px 5px -1px rgba(0, 0, 0, 0.2),0px 5px 8px 0px rgba(0, 0, 0, 0.14),0px 1px 14px 0px rgba(0, 0, 0, 0.12)",
          "0px 3px 5px -1px rgba(0, 0, 0, 0.2),0px 6px 10px 0px rgba(0, 0, 0, 0.14),0px 1px 18px 0px rgba(0, 0, 0, 0.12)",
          "0px 4px 5px -2px rgba(0, 0, 0, 0.2),0px 7px 10px 1px rgba(0, 0, 0, 0.14),0px 2px 16px 1px rgba(0, 0, 0, 0.12)",
          "0px 5px 5px -3px rgba(0, 0, 0, 0.2),0px 8px 10px 1px rgba(0, 0, 0, 0.14),0px 3px 14px 2px rgba(0, 0, 0, 0.12)",
          "0px 5px 6px -3px rgba(0, 0, 0, 0.2),0px 9px 12px 1px rgba(0, 0, 0, 0.14),0px 3px 16px 2px rgba(0, 0, 0, 0.12)",
          "0px 6px 6px -3px rgba(0, 0, 0, 0.2),0px 10px 14px 1px rgba(0, 0, 0, 0.14),0px 4px 18px 3px rgba(0, 0, 0, 0.12)",
          "0px 6px 7px -4px rgba(0, 0, 0, 0.2),0px 11px 15px 1px rgba(0, 0, 0, 0.14),0px 4px 20px 3px rgba(0, 0, 0, 0.12)",
          "0px 7px 8px -4px rgba(0, 0, 0, 0.2),0px 12px 17px 2px rgba(0, 0, 0, 0.14),0px 5px 22px 4px rgba(0, 0, 0, 0.12)",
          "0px 7px 8px -4px rgba(0, 0, 0, 0.2),0px 13px 19px 2px rgba(0, 0, 0, 0.14),0px 5px 24px 4px rgba(0, 0, 0, 0.12)",
          "0px 7px 9px -4px rgba(0, 0, 0, 0.2),0px 14px 21px 2px rgba(0, 0, 0, 0.14),0px 5px 26px 4px rgba(0, 0, 0, 0.12)",
          "0px 8px 9px -5px rgba(0, 0, 0, 0.2),0px 15px 22px 2px rgba(0, 0, 0, 0.14),0px 6px 28px 5px rgba(0, 0, 0, 0.12)",
          "0px 8px 10px -5px rgba(0, 0, 0, 0.2),0px 16px 24px 2px rgba(0, 0, 0, 0.14),0px 6px 30px 5px rgba(0, 0, 0, 0.12)",
          "0px 8px 11px -5px rgba(0, 0, 0, 0.2),0px 17px 26px 2px rgba(0, 0, 0, 0.14),0px 6px 32px 5px rgba(0, 0, 0, 0.12)",
          "0px 9px 11px -5px rgba(0, 0, 0, 0.2),0px 18px 28px 2px rgba(0, 0, 0, 0.14),0px 7px 34px 6px rgba(0, 0, 0, 0.12)",
          "0px 9px 12px -6px rgba(0, 0, 0, 0.2),0px 19px 29px 2px rgba(0, 0, 0, 0.14),0px 7px 36px 6px rgba(0, 0, 0, 0.12)",
          "0px 10px 13px -6px rgba(0, 0, 0, 0.2),0px 20px 31px 3px rgba(0, 0, 0, 0.14),0px 8px 38px 7px rgba(0, 0, 0, 0.12)",
          "0px 10px 13px -6px rgba(0, 0, 0, 0.2),0px 21px 33px 3px rgba(0, 0, 0, 0.14),0px 8px 40px 7px rgba(0, 0, 0, 0.12)",
          "0px 10px 14px -6px rgba(0, 0, 0, 0.2),0px 22px 35px 3px rgba(0, 0, 0, 0.14),0px 8px 42px 7px rgba(0, 0, 0, 0.12)",
          "0px 11px 14px -7px rgba(0, 0, 0, 0.2),0px 23px 36px 3px rgba(0, 0, 0, 0.14),0px 9px 44px 8px rgba(0, 0, 0, 0.12)",
          "0px 11px 15px -7px rgba(0, 0, 0, 0.2),0px 24px 38px 3px rgba(0, 0, 0, 0.14),0px 9px 46px 8px rgba(0, 0, 0, 0.12)",
        ],
        "shape": Object {
          "borderRadius": 4,
        },
        "spacing": Object {
          "unit": 8,
        },
        "stepperDotsStyle": Object {
          "disabled": Object {
            "visibility": "hidden",
          },
          "dot": Object {
            "background": "white",
            "margin": "0px 6px 0px 6px",
            "transform": "scale(1.2)",
          },
          "dotActive": Object {
            "background": "radial-gradient(#1D585D 0, #1D585D 35%, white 50%, white 100%)",
          },
          "label": Object {
            "color": "white",
          },
          "root": Object {
            "background": "#5EB8C0",
            "justifyContent": "space-evenly",
          },
        },
        "styles": Object {
          "classes": Object {
            "body1": "f5 roboto-light gray_01",
            "body2": "f4 roboto-light gray_01",
            "body4": "f5 roboto-medium gray_01",
            "button": "f6 roboto-medium pink toUppercase",
            "highlight1": "f5 roboto-medium pink",
            "highlight2": "f6 roboto-regular gray_01",
            "highlight3": "f5 roboto-light pink",
            "links": "f6 roboto-bold gray_02",
            "subtitle1": "f3 roboto-light gray_01",
            "subtitle2": "f3 roboto-medium turquise",
            "subtitle3": "f4 brandon-regular turquise",
            "subtitle4": "f6 brandon-bold turquise",
            "subtitle5": "f7 roboto-light gray_04",
            "title1": "f4 brandon-regular pink",
            "title2": "f1 roboto-medium gray_01",
            "title3": "f1 roboto-medium turquise",
          },
          "colors": Object {
            "aqua": "#8ACAC7",
            "bgFooter": "#383838",
            "cobalt": "#007CA3",
            "fairy": "#B6DDDC",
            "gray_01": "#636363",
            "gray_02": "#9C9C9C",
            "gray_03": "#D4D4D4",
            "gray_04": "#F0F0F0",
            "green": "#94BA1D",
            "pink": "#D4436B",
            "purple": "#632B7D",
            "see": "#347A7C",
            "seeGradient": "linear-gradient(26deg, #347A7C, #8ACAC7)",
            "siren": "#019193",
            "turquise": "#5EB8C0",
            "white": "#ffffff",
          },
          "fonts": Object {
            "body1": Object {
              "color": "#636363",
              "fontFamily": "'Roboto Light', sans-serif",
              "fontSize": "1rem",
            },
            "body2": Object {
              "color": "#9C9C9C",
              "fontFamily": "'Roboto Regular', sans-serif",
              "fontSize": ".875rem",
            },
            "body3": Object {
              "color": "#5EB8C0",
              "fontFamily": "'Roboto Medium', sans-serif",
              "fontSize": "3rem",
            },
            "body4": Object {
              "color": "#636363",
              "fontFamily": "'Roboto Medium', sans-serif",
              "fontSize": "1rem",
            },
            "button": Object {
              "color": "#D4436B",
              "fontFamily": "'Roboto Medium', sans-serif",
              "fontSize": ".875rem",
            },
            "footerText": Object {
              "color": "#F0F0F0",
              "fontFamily": "Roboto-Light, sans-serif",
              "fontSize": "0.875rem",
            },
            "headline": Object {
              "color": "#636363",
              "fontFamily": "Roboto-Light, sans-serif",
              "fontSize": "1.25rem",
            },
            "highlight1": Object {
              "color": "#D4436B",
              "fontFamily": "'Roboto Medium', sans-serif",
              "fontSize": "1rem",
            },
            "highlight2": Object {
              "color": "#636363",
              "fontFamily": "'Roboto Regular', sans-serif",
              "fontSize": ".875rem",
            },
            "highlight3": Object {
              "color": "#D4436B",
              "fontFamily": "'Roboto Light', sans-serif",
              "fontSize": "1rem",
            },
            "links": Object {
              "color": "#D4436B",
              "fontFamily": "'Brandon Regular', sans-serif",
              "fontSize": "1.25rem",
            },
            "mainTitle": Object {
              "color": "#ffffff",
              "fontFamily": "'Brandon Light', sans-serif",
              "fontSize": "1.6rem",
            },
            "subheading": Object {
              "color": "#D4436B",
              "fontFamily": "'Roboto Medium', sans-serif",
              "fontSize": "1rem",
            },
            "subtitle1": Object {
              "color": "#636363",
              "fontFamily": "'Roboto Light', sans-serif",
              "fontSize": "1.5rem",
            },
            "subtitle2": Object {
              "color": "#5EB8C0",
              "fontFamily": "'Roboto Medium', sans-serif",
              "fontSize": "1.5rem",
            },
            "subtitle3": Object {
              "color": "#5EB8C0",
              "fontFamily": "'Roboto Regular', sans-serif",
              "fontSize": "1.25rem",
            },
            "subtitle4": Object {
              "color": "#5EB8C0",
              "fontFamily": "'Roboto Regular', sans-serif",
              "fontSize": ".875rem",
            },
            "title1": Object {
              "color": "#D4436B",
              "fontFamily": "'Brandon Regular', sans-serif",
              "fontSize": "1.25rem",
            },
            "title2": Object {
              "color": "#636363",
              "fontFamily": "'Roboto Medium', sans-serif",
              "fontSize": "3rem",
            },
            "title3": Object {
              "color": "#5EB8C0",
              "fontFamily": "'Roboto Medium', sans-serif",
              "fontSize": "3rem",
            },
          },
        },
        "transitions": Object {
          "create": [Function],
          "duration": Object {
            "complex": 375,
            "enteringScreen": 225,
            "leavingScreen": 195,
            "short": 250,
            "shorter": 200,
            "shortest": 150,
            "standard": 300,
          },
          "easing": Object {
            "easeIn": "cubic-bezier(0.4, 0, 1, 1)",
            "easeInOut": "cubic-bezier(0.4, 0, 0.2, 1)",
            "easeOut": "cubic-bezier(0.0, 0, 0.2, 1)",
            "sharp": "cubic-bezier(0.4, 0, 0.6, 1)",
          },
          "getAutoHeightDuration": [Function],
        },
        "typography": Object {
          "body1": Object {
            "color": "#636363",
            "fontFamily": "'Roboto Light', sans-serif",
            "fontSize": "1rem",
            "fontWeight": 400,
            "lineHeight": "1.46429em",
          },
          "body1Next": Object {
            "color": "rgba(0, 0, 0, 0.87)",
            "fontFamily": "'Roboto Light', 'sans-serif'",
            "fontSize": "NaNrem",
            "fontWeight": 400,
            "lineHeight": 1.5,
          },
          "body2": Object {
            "color": "#9C9C9C",
            "fontFamily": "'Roboto Regular', sans-serif",
            "fontSize": ".875rem",
            "fontWeight": 500,
            "lineHeight": "1.71429em",
          },
          "body2Next": Object {
            "color": "rgba(0, 0, 0, 0.87)",
            "fontFamily": "'Roboto Light', 'sans-serif'",
            "fontSize": "NaNrem",
            "fontWeight": 400,
            "lineHeight": 1.5,
          },
          "button": Object {
            "color": "#D4436B",
            "fontFamily": "'Roboto Medium', sans-serif",
            "fontSize": ".875rem",
            "fontWeight": 500,
            "textTransform": "uppercase",
          },
          "buttonNext": Object {
            "color": "rgba(0, 0, 0, 0.87)",
            "fontFamily": "'Roboto Light', 'sans-serif'",
            "fontSize": "NaNrem",
            "fontWeight": 500,
            "lineHeight": 1.5,
            "textTransform": "uppercase",
          },
          "caption": Object {
            "color": "rgba(0, 0, 0, 0.54)",
            "fontFamily": "'Roboto Light', 'sans-serif'",
            "fontSize": "NaNrem",
            "fontWeight": 400,
            "lineHeight": "1.375em",
          },
          "captionNext": Object {
            "color": "rgba(0, 0, 0, 0.87)",
            "fontFamily": "'Roboto Light', 'sans-serif'",
            "fontSize": "NaNrem",
            "fontWeight": 400,
            "lineHeight": 1.66,
          },
          "color": "#636363",
          "display1": Object {
            "color": "#636363",
            "fontFamily": "'Roboto Medium', sans-serif",
            "fontSize": "3rem",
            "fontWeight": 400,
            "lineHeight": "1.20588em",
          },
          "display2": Object {
            "color": "#5EB8C0",
            "fontFamily": "'Roboto Medium', sans-serif",
            "fontSize": "3rem",
            "fontWeight": 400,
            "lineHeight": "1.13333em",
            "marginLeft": "-.02em",
          },
          "display3": Object {
            "color": "rgba(0, 0, 0, 0.54)",
            "fontFamily": "'Roboto Light', 'sans-serif'",
            "fontSize": "NaNrem",
            "fontWeight": 400,
            "letterSpacing": "-.02em",
            "lineHeight": "1.30357em",
            "marginLeft": "-.02em",
          },
          "display4": Object {
            "color": "rgba(0, 0, 0, 0.54)",
            "fontFamily": "'Roboto Light', 'sans-serif'",
            "fontSize": "NaNrem",
            "fontWeight": 300,
            "letterSpacing": "-.04em",
            "lineHeight": "1.14286em",
            "marginLeft": "-.04em",
          },
          "fontFamily": "'Roboto Light', 'sans-serif'",
          "fontSize": "1rem",
          "fontWeightLight": 300,
          "fontWeightMedium": 500,
          "fontWeightRegular": 400,
          "h1": Object {
            "color": "rgba(0, 0, 0, 0.87)",
            "fontFamily": "'Roboto Light', 'sans-serif'",
            "fontSize": "NaNrem",
            "fontWeight": 300,
            "lineHeight": 1,
          },
          "h2": Object {
            "color": "rgba(0, 0, 0, 0.87)",
            "fontFamily": "'Roboto Light', 'sans-serif'",
            "fontSize": "NaNrem",
            "fontWeight": 300,
            "lineHeight": 1,
          },
          "h3": Object {
            "color": "rgba(0, 0, 0, 0.87)",
            "fontFamily": "'Roboto Light', 'sans-serif'",
            "fontSize": "NaNrem",
            "fontWeight": 400,
            "lineHeight": 1.04,
          },
          "h4": Object {
            "color": "rgba(0, 0, 0, 0.87)",
            "fontFamily": "'Roboto Light', 'sans-serif'",
            "fontSize": "NaNrem",
            "fontWeight": 400,
            "lineHeight": 1.17,
          },
          "h5": Object {
            "color": "rgba(0, 0, 0, 0.87)",
            "fontFamily": "'Roboto Light', 'sans-serif'",
            "fontSize": "NaNrem",
            "fontWeight": 400,
            "lineHeight": 1.33,
          },
          "h6": Object {
            "color": "rgba(0, 0, 0, 0.87)",
            "fontFamily": "'Roboto Light', 'sans-serif'",
            "fontSize": "NaNrem",
            "fontWeight": 500,
            "lineHeight": 1.6,
          },
          "headline": Object {
            "color": "#636363",
            "fontFamily": "Roboto-Light, sans-serif",
            "fontSize": "1.25rem",
            "fontWeight": 400,
            "lineHeight": "1.35417em",
          },
          "overline": Object {
            "color": "rgba(0, 0, 0, 0.87)",
            "fontFamily": "'Roboto Light', 'sans-serif'",
            "fontSize": "NaNrem",
            "fontWeight": 400,
            "lineHeight": 2.66,
            "textTransform": "uppercase",
          },
          "pxToRem": [Function],
          "round": [Function],
          "subheading": Object {
            "color": "#D4436B",
            "fontFamily": "'Roboto Medium', sans-serif",
            "fontSize": "1rem",
            "fontWeight": 400,
            "lineHeight": "1.5em",
          },
          "subtitle1": Object {
            "color": "rgba(0, 0, 0, 0.87)",
            "fontFamily": "'Roboto Light', 'sans-serif'",
            "fontSize": "NaNrem",
            "fontWeight": 400,
            "lineHeight": 1.75,
          },
          "subtitle2": Object {
            "color": "rgba(0, 0, 0, 0.87)",
            "fontFamily": "'Roboto Light', 'sans-serif'",
            "fontSize": "NaNrem",
            "fontWeight": 500,
            "lineHeight": 1.57,
          },
          "title": Object {
            "color": "#D4436B",
            "fontFamily": "'Brandon Regular', sans-serif",
            "fontSize": "1.25rem",
            "fontWeight": 500,
            "lineHeight": "1.16667em",
          },
          "useNextVariants": false,
        },
        "zIndex": Object {
          "appBar": 1100,
          "drawer": 1200,
          "mobileStepper": 1000,
          "modal": 1300,
          "snackbar": 1400,
          "tooltip": 1500,
        },
      }
    }
  >
    <WithStyles(MktAppsFooter)>
      <WithStyles(MktAppsFooter) />
    </WithStyles(MktAppsFooter)>
  </MuiThemeProvider>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "children": <WithStyles(MktAppsFooter) />,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {},
      "ref": null,
      "rendered": null,
      "type": [Function],
    },
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": <WithStyles(MktAppsFooter) />,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
